travis/script_deploy_android_snapshot.sh

```diff
@@ -3,7 +3,16 @@
 set -e
 set -o pipefail
 
-if [ "${PLATFORM}" = "android" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then
+if [ "${PLATFORM}" = "android" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "mult-android-arch" ]; then
+    
```

Comment:
I had it there for testing, but I guess for now we want to build all (or selected architectures) on every master build. So before merging this, we can revert this change back to `master`.